packages/core/src/sdk/overrides/PageProvider.tsx

```diff
@@ -23,13 +23,15 @@ export interface PLPContext extends PageGlobalContext {
     ClientProductGalleryQueryQuery & {
       pages: ClientManyProductsQueryQuery[]
     }
+  globalSectionsSettings?: Record<string, any>
 }
 
 export interface SearchPageContext extends PageGlobalContext {
   data?: SearchPageContextType &
     ClientProductGalleryQueryQuery & {
       pages: ClientManyProductsQueryQuery[]
     }
+  globalSectionsSettings?: Record<string, any>
```

Comment:
fix: those `globalSectionsSettings` are outdated
```suggestion
}

export interface SearchPageContext extends PageGlobalContext {
  data?: SearchPageContextType &
    ClientProductGalleryQueryQuery & {
      pages: ClientManyProductsQueryQuery[]
    }
```